internal/services/aws_reservation_service.go

```diff
@@ -126,10 +126,10 @@ func CreateAWSReservation(w http.ResponseWriter, r *http.Request) {
 	}
 
 	launchJob := worker.Job{
-		Type:     jobs.TypeLaunchInstanceAws,
-		Identity: identity,
+		Type:      jobs.TypeLaunchInstanceAws,
+		Identity:  id,
+		AccountID: accountId,
```

Comment:
The `Enqueue` function could theoretically set identity and account id automatically from the context when it is blank (or zero). But this is fine.